week04/code/LinkedListTester.cs

```diff
@@ -41,6 +53,14 @@ public static class LinkedListTester {
         ll.Remove(2);
         Console.WriteLine(ll.ToString()); // <LinkedList>{4, 35, 2, 2}
 
+        var ll4 = new LinkedList();
+        ll4.Remove(0);
+        Console.WriteLine(ll4.ToString()); // <LinkedList>{}
+        ll4.InsertHead(2);
+        ll4.Remove(2);
+        Console.WriteLine(ll4.ToString()); // <LinkedList>{}
+        Console.WriteLine(ll4.HeadAndTailAreNull()); // True
+
```

Comment:
Catches `Remove` not working properly on an empty list or a list with 1 element.